content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -219,9 +220,29 @@ Run:
 kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
 ```
 
-### Pipeline samples that require "user-gcp-sa" secret
+### Securing the cluster with fine-grained GCP permission control
```

Comment:
Hallo @Bobgy This looks great! One thing though: We need to separate out the GCP-specific docs from the generic Pipelines doc that's applicable to all platforms. See issue https://github.com/kubeflow/website/issues/1253

For that reason, I recommend that you move this new section (securing a cluster) onto a new page in the [Kubeflow GCP docs](https://www.kubeflow.org/docs/gke/), rather than adding more GCP-specific info to the current page.

Note: Sometime soon I plan to add a new section to the [Kubeflow GCP docs](https://www.kubeflow.org/docs/gke/) called **Kubeflow Pipelines on GCP**. I'll move the existing [Pipelines E2E tutorial](https://www.kubeflow.org/docs/gke/pipelines-tutorial/) to the new section, as well as the new doc you create, and also some other pieces of content that are specific to Pipelines on GCP.

Reply:
Thanks! I've moved these to a new page.